dotnet/src/Connectors/Connectors.AzureOpenAI/Core/ClientCore.TextToAudio.cs

```diff
@@ -64,4 +70,12 @@ private static (GeneratedSpeechFormat Format, string MimeType) GetGeneratedSpeec
             "PCM" => (GeneratedSpeechFormat.Pcm, "audio/l16"),
             _ => throw new NotSupportedException($"The format '{format}' is not supported.")
         };
+
+    private string GetModelId(AzureOpenAITextToAudioExecutionSettings executionSettings, string? modelId)
+    {
+        return
+            !string.IsNullOrWhiteSpace(modelId) ? modelId! :
+            !string.IsNullOrWhiteSpace(executionSettings.ModelId) ? executionSettings.ModelId! :
+            this.DeploymentOrModelName;
```

Comment:
On OpenAI V2 I'm using ModelId, not name, suggest making this update too here or later.

```suggestion
            this.DeploymentOrModelId;
```

Reply:
Yep, I'll do it in a separate PR later.